bluebutton-data-model-codebook-extractor/src/test/java/gov/hhs/cms/bluebutton/data/codebook/extractor/SupportedCodebookTest.java

```diff
@@ -69,7 +69,7 @@ public void findDuplicateVariables() throws IOException {
 		Assert.assertTrue(duplicatedVariableIds.contains("GNDR_CD"));
 
 		// Blow up if anything more than those known problems appears.
-		Assert.assertEquals(3, duplicatedVariableIds.size());
+		Assert.assertEquals(87, duplicatedVariableIds.size());
```

Comment:
I originally wrote this code with the intent of ensuring that we didn't run into bad "surprises" with the codebooks, like, e.g. two variables with the same name but different definitions.

I think this section (and the one right above it) need a bit of rework. Any thoughts on what we might do with it?